nexus/src/app/background/services_config.rs

```diff
@@ -0,0 +1,378 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
To the "service table":

- Add a global tracking service generation table
- Add a "added in" and "removed from" value in the service table